janus_core/single_point.py

```diff
@@ -327,7 +340,7 @@ def run_single_point(
         if "stress" in properties or len(properties) == 0:
             results["stress"] = self._get_stress()
 
-        self._clean_results(properties=properties)
+        self._clean_results(results, properties=properties)
```

Comment:
add a note above this saying why we need to call it... cause may look odd to someone who checks the code.